Assets/_Application/Projects/ProjectDataHandler.cs

```diff
@@ -191,26 +190,22 @@ private void LoadDefaultProject()
             credentialHandler.SetUri(url);
             credentialHandler.ApplyCredentials();
 #else
-            var filePath = Path.Combine(Application.streamingAssetsPath, defaultProjectFileName);
+            var filePath = Path.Combine(Application.persistentDataPath, Application.streamingAssetsPath, defaultProjectFileName);
```

Comment:
This is not possible - the streamingAssetsPath is a complete path that is not cumulative with the persistentDataPath. streamingAssetsPath points to a folder on the webserver itself